modules/nf-core/gnu/split/meta.yml

```diff
@@ -33,15 +33,15 @@ output:
           description: |
             Groovy Map containing sample information
             e.g. `[ id:'sample1', single_end:false ]`
-      - ' "*split*" ':
+      - ' "${outfile_prefix}.*" ':
           type: file
           description: Split files
-          pattern: "*split*"
+          pattern: "${outfile_prefix}.*"
   - versions:
       - versions.yml:
           type: file
-          description: The split files
+          description: File containing software versions
+          pattern: "versions.yml"
 authors:
   - "@k1sauce"
-maintainers:
```

Comment:
Was `maintainers` deleted on purpose? If so, you should delete the github handle, it is duplicated now

Reply:
No problem, adding it back in now. 

Only issue now is ULTRA_PIPELINE